examples/logger.py

```diff
@@ -3,13 +3,38 @@
 """
 
 import time
+import logging
 from typing import Any, Optional, Tuple
 
 from PyQt5.QtCore import QObject, pyqtSlot, pyqtSignal
 from PyQt5.QtWidgets import QWidget, QVBoxLayout, QPushButton, QTextEdit, QLabel, QDialogButtonBox
 
 from qiwis import BaseApp
 
+class Signaller(QObject):
+    """Signal for LoggingHandler"""
+    signal = pyqtSignal(str)
+
+class LoggingHandler(QObject, logging.Handler):
+    """Handler for logger.
+
+    Sends log message to connected function using emit
+
+    Signal:
+          signal: Helps to send message to destination.
+    """
+    signal = pyqtSignal(str)
+
+    def __init__(self, slotfunc, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
First of all, please add the constructor docstring about arguments.
Also are both `args` and `kwargs` necessary?